src/setup/demo.md

```diff
@@ -0,0 +1,352 @@
+# Hands-on Substra demo
+
+Check this [page](https://www.substra.ai/en/demo) and get in touch with us, we will then provide you with instructions and credentials!
+
+## 0. Pre-requisites & Overview of the demo
+
+### Terminology
+
+To follow this tutorial for using the demo instance, it is assumed you have read the documentation of the Substra Framework and are familiar with its core concepts. In particular the following terms: organisation, asset, algo, objective, metric, opener, train task, test task.
```

Comment:
Seems a little bit heavy to must read "all" documentation. but why not.

Reply:
Indeed, cc @bowni 
I'll add the link to the glossary here, by the way